runtime/lua/vim/_defaults.lua

```diff
@@ -65,6 +65,11 @@ do
     desc = ':help CTRL-L-default',
   })
 
+  --- https://github.com/tpope/vim-scriptease style mapping for `:Inspect`.
+  ---
+  --- See |zS-default|
+  vim.keymap.set('n', 'zS', '<Cmd>Inspect<CR>', { desc = ':help zS-default' })
```

Comment:
nit/suggestion: shouldn't make a difference, but perhaps more idiomatic?

```suggestion
  vim.keymap.set('n', 'zS', vim.show_pos, { desc = ':help zS-default' })
```